lib/libc/mips/gen/_ctx_start.S

```diff
@@ -51,6 +51,7 @@ __FBSDID("$FreeBSD$");
  * stack frame setup by makecontext() for the callback function.
  */
 ENTRY(_ctx_start)
+	.cfi_startproc
```

Comment:
Does this not belong in ENTRY?

Reply:
Also... why on earth does ASM_ENTRY exist... it's just the same but also sets the type to `@function`...

Reply:
It appeared that this was also used for nested labels rather than just start of function, but let me double check.

Reply:
Oh, right, it drops the `.ent`, but isn't that what EENTRY is meant to be in other ports?

Reply:
I'm pretty sure mips just has ENTRY backwards, and that it's currently what EENTRY should be, with ASM_ENTRY meant to be the normal ENTRY. Most of sys/mips seems to use them that way... (the *only* use of ASM_ENTRY is in locore for _start......).

Reply:
Seems to work just fine with ENTRY changed. Thanks for pointing it out.